App/Views/roles/edit.view.php

```diff
@@ -0,0 +1,70 @@
+<?php
+require "../App/Views/header.view.php";
+?>
+
+<form action="/roles/<?php echo $role->getId(); ?>/update" method="post">
+    <div class="form-group">
+    		<label for="name">Name: </label>
+    		<input type="text" name="name" class="form-control" id="name" value="<?php echo $role->getName(); ?>" required>
+  	</div>
+
+    <div class="form-group">
+      <label class="checkbox-inline">
+          <input type="checkbox"
+             data-toggle="toggle"
+             id="MANAGE_ROLES"
+             name="MANAGE_ROLES"
+             <?php if (isset($permissions['MANAGE_ROLES'])) echo 'checked'?>
+             value=true> Manage roles
+      </label>
+    </div>
+    <div class="form-group">
+      <label class="checkbox-inline">
+          <input type="checkbox"
+             data-toggle="toggle"
+             id="MANAGE_TEAM_LEADERS"
+             name="MANAGE_TEAM_LEADERS"
+             <?php if (isset($permissions['MANAGE_TEAM_LEADERS'])) echo 'checked'?>
+             value=true> Manage Team Leaders
+      </label>
+    </div>
+    <div class="form-group">
+      <label class="checkbox-inline">
+          <input type="checkbox"
+             data-toggle="toggle"
+             id="MANAGE_USERS"
+             name="MANAGE_USERS"
+             <?php if (isset($permissions['MANAGE_USERS'])) echo 'checked'?>
+             value=true> Manage users
+      </label>
+    </div>
+    <div class="form-group">
+      <label class="checkbox-inline">
+          <input type="checkbox"
+             data-toggle="toggle"
+             id="MANAGE_TASKS"
```

Comment:
You can fetch all the permissions from the Enum and create a for here to render all permissions.